### ISSUE TITLE
Outreachy Documentation Project: Milena Brito

### ISSUE THREAD

**Issue by srcmilena** [ersilia-os/ersilia#143]
Applicant: Milena Brito
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [x] Create a README file with the name <README_yourname.md> under the /documentation folder
- [x] Link the #PR in a comment under this issue
- [ ] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [ ] Incorporate feedback from the mentor
- [x] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [ ] Incorporate feedback from the mentor
- [x] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [x] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [x] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [ ] Test one model
- [ ] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [x] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [x] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [x] Create one image / slide to explain Ersilia’s mission and vision
- [x] Link to the image/slide as a comment under this issue
- [x] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [x] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [x] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [x] Add your card as a comment to this issue
- [x] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [x] Look up two other projects and comment on their issues with feedback on one of their tasks
- [x] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [x] Add a comment in this issue with your new idea, tagging the mentor 
- [x] Get feedback from the mentor and act accordingly
- [x] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by srcmilena** [comment]
When I opened Outreachy project list and started to seek a project to join, I was looking for something unusual and different from what I study and already do (software development) and at same time, something that I am curious about. 
When I found Ersilia and started reading about their organization itself, I confess that seeing "IA", "Machine Learning" caught all my attention because before this all, I was thinking about where I could ever experience these activities and I found it here. I was so impressed and even more passionate because I really LOVE technology. I started imagining how it could work while I was already looking to join in this amazing project. When I quickly joined, in the last week, I noticed that I was in the right place and I felt that even more when I was in the meeting. It is my pleasure to be here and having this experience. Such an amazing period of discovering new people and learning new things. I appreciate it so much!

**Comment by srcmilena** [comment]
Graphic material

- Created by Canva
  - https://www.canva.com/design/DAE9FKhK0nU/Fec4u92dKzWVT1qzjH4O6Q/view?utm_content=DAE9FKhK0nU&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton


- Details
  - Fonts
    - Open Sans Light
    - Open Sans
    - Open Sans Bold

   - Font size
     - 24 (subtitles in "Vision" topics)
     - 26 ("Vision" text)
     - 27 ("Ersilia" and "Mission" text)
     - 30 (titles)


- Sources
  - https://ersilia.gitbook.io/ersilia-wiki/diversity-and-inclusion-statement
  - https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5

**Comment by GemmaTuron** [comment]
Hi @srcmilena 

Thanks for the work! I really like your background. can I give you a few suggestions?
The Ersilia metaphor has two sides, one, abandoned cities and abandoned research, as you well describe. The other more positive one is to establish threads and a network of communications between scientists. Can you rephrase to show this idea? To give an encouraging message!
The mission of the initiative is to support research into infectious and neglected diseases, with the vision to provide more equal access to healthcare. Can you rephrase the statements thinking on that?
The backgrpund color is black or the Ersilia Plum? we could make it ersilia plum and even perhaps the words in ersilia's green! 

good work!

**Comment by srcmilena** [comment]
Hello, @GemmaTuron,

Thank you so much for your feedback and suggestions. I changed what you said. May you check? Please.

I altered the following things:
- Texts "Ersilia", "Mission" and "Vision";
- Background color;
- Text color;
- Color of the logo.

***

- Created by Canva

  - https://www.canva.com/design/DAE-VWtR2hE/lD74pk0uXhPcIl335PWJqg/view?utm_content=DAE-VWtR2hE&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

- Text details

  - Fonts

    - Open Sans Light;
    - Open Sans;
    - Open Sans Bold.

  - Font size

    - 24 (subtitles in "Vision" topics);
    - 26 ("Vision" text);
    - 27 ("Ersilia" and "Mission" text);
    - 30 (titles).

 - Colors

   - Background color: mint #BEE6B4;
   - Font color: plum #50285A.

- Sources

  - https://ersilia.gitbook.io/ersilia-wiki/diversity-and-inclusion-statement
  - https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5

**Comment by srcmilena** [comment]
# Other

@GemmaTuron 

When I joined the Slack channel/group, I was a little lost and stayed lost until the first meeting took place. I am not blaming anyone as this is due to my lack of knowledge about the platform, project and it was a very pleasant and new experience. I am thankful for that meeting, which happened 1 week after the start of the contributions, because it was from there that I was able to get myself situated and actually start doing something because, before that, I was going the wrong way lol. 

My only suggestion is just to simplify this step by step by a message (if possible), because I believe it would be very useful for those who showed up and were in a situation like mine.

Other than that, I would like to thank you for your attention and helpfulness in this process. You have been great!

**Comment by srcmilena** [comment]
# Writing dissemination material

(Strategic plan 2021-2023)

- Page blogspot created by Google docs
  - https://docs.google.com/document/d/13tRV--Z_OFeTPGl0McgUAP4i1MAwNnkFC08kbPjzAxA/edit?usp=sharing

**Comment by srcmilena** [comment]
# Github documentation
#### README file (I used the strategic plan file)

- README_Milena.md
  - [PR #255 ](https://github.com/ersilia-os/ersilia/pull/255)

**Comment by srcmilena** [comment]
# Technical skills (required for the tutorial only)

I installed but after installing I noticed that is needed at least windows 10 and my currently version is older.
